docs/reference/services.rst

```diff
@@ -0,0 +1,63 @@
+=================
+MicroOVN Services
+=================
+
+This document provides brief explanation of services provided by MicroOVN. They
+are generally not meant for direct user interaction and should be used only
+if necessary. List and status of these services can be displayed by running:
+
+.. code-block:: none
+
+    snap services microovn
+
+microovn.central
+----------------
+
+.. warning::
+
+    Deprecated: This service will be removed in future release
+
+This is a shortcut service. Starting this service will start and enable
```

Comment:
... and enable multiple services:

* ``microovn.ovn-nb``
* ``microovn.ovn-sb``
* ``microovn.ovn-northd``

However, ...